unofficial/c511027651.lua

```diff
@@ -0,0 +1,36 @@
+local s,id=GetID()
+function s.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCategory(CATEGORY_DRAW)

```

Comment:
You still need CATEGORY_TOHAND since you are adding something to your hand other than via drawing